release-notes/WCF-Web-Service-Reference-notes.md

```diff
@@ -1,16 +1,17 @@
-# WCF Web Service Reference Provider
+# WCF Web Service Reference Provider - Release Notes
 
-## Information
-
-* [Usage guide](/Documentation/WCF-Web-Service-Reference-guide.md)
-* [Report an issue](https://github.com/dotnet/wcf/issues/new)
+WCF Web Service Reference connected services provider is now part of Visual Studio 2017. Please refer to [usage guide](/Documentation/WCF-Web-Service-Reference-guide.md)
+for instructions or [report an issue or give us feedback](https://github.com/dotnet/wcf/issues/new).
```

Comment:
for instructions, or [to report an issue, or to give us feedback]
// but you may want to have a native English speaker review this.

Reply:
Yes, I know this paragraph is bit weird. I have it rewritten.